code/game/machinery/computer/records/security.dm

```diff
@@ -154,23 +162,36 @@ CREATION_TEST_IGNORE_SUBTYPES(/obj/machinery/computer/records/security)
 			return TRUE
 
 		if("print_record")
-			print_record(user, target_record, params["alias"], params["desc"], params["head"], params["type"])
+			print_record(user, target_record, sanitize_ic(params["alias"]), sanitize_ic(params["desc"]), sanitize_ic(params["head"]), sanitize_ic(params["type"]))
 			return TRUE
 
 		if("set_note")
-			target_record.set_security_note(params["security_note"])
+			target_record.set_security_note(sanitize_ic(params["security_note"]))
 			return TRUE
 
 		if("set_wanted")
-			target_record.set_wanted_status(params["status"])
-			return TRUE
-
-		if("set_amount")
```

Comment:
removed action

Reply:
It wasn't removed, it was moved up to the top of the proc prior to the authenticated var being checked.